resources/assets/sass/_devices.scss

```diff
@@ -130,3 +135,27 @@ td.col-device-auto {
         top: 2px;
     }
 }
+
+.device-info {
+  display: grid;
```

Comment:
I haven't used display: grid before - I'll have to get up to speed on that - it's browser support is OK?

Reply:
Given what we discussed, yes.  

https://caniuse.com/#feat=css-grid

I'm fairly new to grid, and came via flexbox which is worth learning first.  Both of them are fantastic - it becomes easy to produce layouts which render on variably screen sizes.  In retrospect I wish I'd never learnt Bootstraps column-based layout and gone straight to this.

Reply:
Great :+1: I've used flexbox so will brush up on grid

Reply:
The guy who encouraged me to learn it suggested this, which I found quite useful.  

https://codepip.com/games/grid-garden/